startUp.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import sys
+import subprocess
+import time
+
+import settings
+import classes
+
+def main():
+    with open(sys.argv[1]) as file:
+        solarCarProcesses = [classes.Process(path, subprocess.Popen(path)) for path in file.read().splitlines()]
+
+    # Check processes and respond accordingly
+    while True:
+        for solarCarProcess in solarCarProcesses:
+            if solarCarProcess.process.poll():
+                solarCarProcess.process = subprocess.Popen(solarCarProcess.path)
+                solarCarProcess.timesRestarted += 1
+            if solarCarProcess.timesRestarted > settings.MAX_RESTART:
+                print(solarCarProcess.path, "exceeded", settings.MAX_RESTART, "restarts")
```

Comment:
We should only be printing this message when we're attempting to restart a process, right now we will get this message even if the process is running.